hash/saturn.xml

```diff
@@ -9175,6 +9224,21 @@ Sega logo animation draws offset [VDP2]
 		</part>
 	</software>
 
+	<software name="vfremixk" cloneof="vfremix" supported="no">
+		<description>Virtua Fighter Remix (Korea)</description>
+		<year>1995</year>
+		<publisher>Samsung</publisher>
+		<info name="serial" value="GS-9503J"/>
+		<info name="alt_title" value="버쳐 파이터 리믹스"/>
+		<sharedfeat name="compatibility" value="NTSC-K"/>
```

Comment:
Korea is a middleground region for Saturn, needs to checkout if it boots in Japan and/or USA versions BIOSes.